### PR TITLE
Feature/ar marker

### PR DESCRIPTION
**Pull request by Sven65**
Please note that this does not include refactoring.

### CODE REVIEW

**Comment by nRamstedt** [comment]
built and tested on iphone works! a few details:

- [ ] Add PlannerView as a prefab
- [ ] Add [RequireComponent(CameraRaycast)](https://docs.unity3d.com/ScriptReference/RequireComponent.html) to PlannerView

File: pladdra-app/Assets/Scripts/Components/CameraRaycast.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    public GameObject arMarkerPrefab;
+    GameObject createdARMarker;
+
+
+    void placeSphereAtPhysical() {
+        RaycastHit hit;
+
+
+        if (Physics.Raycast(ARCamera.transform.position, ARCamera.transform.forward, out hit, float.MaxValue)) {
+            ARPlane plane = hit.transform.GetComponent<ARPlane>();
```
Comment (nRamstedt) [inline pladdra-app/Assets/Scripts/Components/CameraRaycast.cs]: Hardcoded raycast hit target, perhaps let the view set the target type. That way some views can target ARPlanes while others can target future GridPlane

Comment (nRamstedt) [reply pladdra-app/Assets/Scripts/Components/CameraRaycast.cs]: Or someother way to set different raycast targets

File: pladdra-app/Assets/Scripts/Components/CameraRaycast.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+            if (plane == null) return;
+
+            if (createdARMarker == null) {
+                createdARMarker = Instantiate(arMarkerPrefab, hit.point, Quaternion.identity);
```
Comment (nRamstedt) [inline pladdra-app/Assets/Scripts/Components/CameraRaycast.cs]: CameraRaycast should only have the purpose of Raycasting, here we are rendering the arMarker which could be moved into Planner View/Controller/Model.

File: pladdra-app/Assets/Scripts/Components/CameraRaycast.cs [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+            } else {
+                createdARMarker.transform.position = hit.point;
```
Comment (nRamstedt) [inline pladdra-app/Assets/Scripts/Components/CameraRaycast.cs]: Transform should happen within PlannerView

**Review by nRamstedt — CHANGES_REQUESTED** [review]
built and tested on iphone works! a few details:

- [x] Add PlannerView as a prefab
- [x] Add RequireComponent(CameraRaycast) to PlannerView

File: pladdra-app/Assets/Scripts/Components/CameraRaycast.cs (hunk 1 above)
Comment (Sven65) [reply pladdra-app/Assets/Scripts/Components/CameraRaycast.cs]: I've added a field to the `CameraRaycast` component that allows for this now.

File: pladdra-app/Assets/Scripts/Components/CameraRaycast.cs (hunk 2 above)
Comment (Sven65) [reply pladdra-app/Assets/Scripts/Components/CameraRaycast.cs]: This has been fixed.

**Review by nRamstedt — APPROVED** [review]

**Pull request merged**